esm/deserialize.js

```diff
@@ -1,11 +1,13 @@
+import { b64decode, b64encode } from './b64.js'
 import {
   VOID, PRIMITIVE,
-  ARRAY, OBJECT,
+  ARRAY, OBJECT, BUFFER,
   DATE, REGEXP, MAP, SET,
   ERROR, BIGINT
 } from './types.js';
 
 const env = typeof self === 'object' ? self : globalThis;
+const TypedArray = Reflect.getPrototypeOf(Uint8Array)
```

Comment:
we have `Object` in this scope already, mind using that instead of `Reflect` ? you know, gzip and stuff :-)

Reply:
sure!